src/services/Help.re

```diff
@@ -1,24 +1,44 @@
-let mayday = (text, user, isAdmin, sendMessage, sendMessageWithAttachments) => {
-  Js.Promise.(
-    isAdmin ?
-      sendMessage("Hey, this is restricted to students!") |> ignore :
-      user
-      |> Database.hasUnfinishedHelpItem
-      |> then_(hasUnfinished => {
-           hasUnfinished ?
-             sendMessage("You're already on the help list!") |> ignore :
-             Database.addHelpItem(user, text, sendMessageWithAttachments);
+open Js.Promise;
 
-           resolve();
-         })
-      |> ignore
-  );
-};
+let mayday = (text, user, isAdmin, sendMessage, sendMessageWithAttachments) =>
+  isAdmin ?
+    sendMessage("Hey, this is restricted to students!") |> ignore :
+    user
+    |> Database.hasUnfinishedHelpItem
+    |> then_(hasUnfinished => {
+         hasUnfinished ?
+           sendMessage("You're already on the help list!") |> ignore :
+           Database.addHelpItem(user, text, sendMessageWithAttachments);
 
-let selectRoom = (selectOption, itemId, sendMessage) => {
-  switch (selectOption) {
+         resolve();
+       })
+    |> ignore;
+
+let selectRoom = (selectedOption, itemId, sendMessage) =>
+  switch (selectedOption) {
   | Some(room) =>
-    Database.addRoom(int_of_string(itemId), List.hd(room), sendMessage)
+    Database.addRoom(int_of_string(itemId), List.hd(room))
+    |> then_(roomWasAdded => {
+         roomWasAdded ?
+           Database.getQueuePosition("Joakim1")
+           |> then_(queuePosition => {
+                let msg =
+                  switch (queuePosition) {
+                  | Some(queueNum) =>
+                    "Ok, have a seat and relax. You’re number"
+                    ++ string_of_int(queueNum)
+                    ++ "in line. :coffee:"
+                  | None => "Oops. Couldn't get your position in queue."
+                  };
+
+                sendMessage(msg) |> ignore;
+
+                resolve();
+              })
+           |> ignore :
+           sendMessage("Something went wrong. Try again later!") |> ignore;
```

Comment:
Would be nicer with a more descriptive message here as well I think? Also for debugging purposes, what went wrong?